source/reference/options.txt

```diff
@@ -477,6 +477,87 @@ Session Options
 Client-Side Field Level Encryption Options
 ------------------------------------------
 
-For information on Client-Side Field Level Encryption Options,
-refer to the :manual:`MongoDB Manual
-</reference/program/mongo/#client-side-field-level-encryption-options>`.
+.. option:: --awsAccessKeyId <string>
+
+   An `AWS Access Key 
+   <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html>`__
+   associated with an IAM user who has ``List`` and ``Read`` permissions 
+   for the AWS Key Management Service (KMS). :program:`mongosh` uses the 
+   specified :option:`--awsAccessKeyId` to access the KMS. 
+
+   :option:`--awsAccessKeyId` is required to enable 
+   :ref:`manual-csfle-feature` for the :program:`mongosh` shell session. 
+   :option:`--awsAccessKeyId` requires *both* of the following command 
+   line options:
+
+   - :option:`--awsSecretAccessKey` 
+   - :option:`--keyVaultNamespace`
+
+   If :option:`--awsAccessKeyId` is omitted, use the :method:`Mongo()` 
+   constructor within the shell session to enable client-side field 
+   level encryption.
+
+   To mitigate the risk of leaking access keys into logs, consider 
+   specifying an environmental variable to :option:`--awsAccessKeyId`. 
+
+.. option:: --awsSecretAccessKey <string>
+
+   An AWS `Secret Key <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html>`__
```

Comment:
This is a small thing I noticed, looks like we don't include `AWS` in the link here, while we do include `AWS` in the link in the description for `--awsAccessKeyId`. Should we go for one option to keep it consistent? Also fine as is.